### ISSUE TITLE
tckstats crashes if track file is empty

### ISSUE THREAD

**Issue by LeeReidCSIRO** [MRtrix3/mrtrix3#1322]
Hi,

tckstats will crash if a file provided has no tracks. 

to reproduce:
```
tckgen fod.mif.gz empty.tck -select 0 -seeds 1 -seed_image roi1.nii.gz -include roi2_it_wont_reach.nii.gz

tckstats empty.tck

```
output:
_tckstats: [SYSTEM FATAL CODE: SIGSEGV (11)] Segmentation fault: Invalid memory access_

Apologies if this is fixed in the up to date version: I'm locked into a few-months-old version at the moment but did search for the issue here and didn't find it.

tckinfo seems to work fine.

Cheers,
Lee

**Comment by Lestropie** [comment]
@LeeReidCSIRO: Another good pickup, thanks for the report :+1:

The problem's there even in the development branch, so it's not just an out-of-date installation. It's just not the sort of command I'd spend a lot of time on trying to think of all possible use cases to cover off. #1326 fixes the issue for me, and should hopefully get onto `dev` and then `master` in fairly quick succession.

**Comment by Lestropie** [comment]
Closed by #1326.
